src/Shared/ParameterBindingMethodCache.cs

```diff
@@ -98,6 +99,15 @@ public bool HasTryParseMethod(Type type)
                 };
             }
 
+            if (type == typeof(Uri))
+            {
+                return (expression, formatProvider) => Expression.Call(
```

Comment:
This looks like a reasonable default to me. If people want to customize how URIs are parsed, they can wrap it in another type.

Can you add a comment saying that this is what the `TypeConverter` does as well? This matches the MVC behavior (since it uses TypeConverters)

Reply:
Added a comment regarding `UriKind.RelativeOrAbsolute` choice.
https://github.com/dotnet/runtime/blob/3c33b4293107c607ae8669545114d6f9891432cb/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/UriTypeConverter.cs#L50